Builds.md

```diff
@@ -287,6 +288,8 @@ Build jobs for releases must be executed in proper order on the build server, i.
      * Version Number
      * Update Site URL
      * Supported Eclipse Release(s)
+1. As soon as maven central is updated - send notifications
+1. Add / Update xtext-reference-projects
```

Comment:
thus does not completely hold true (maybe relevant for m1 and release)

Reply:
Done.